templates/nginx.conf.j2

```diff
@@ -28,6 +28,17 @@ http {
         include {{ nginx_conf_dir }}/sites-enabled/*;
 }
 
+{% if nginx_official_repo_mainline %}
```

Comment:
I will prefer to check that ```nginx_stream_params``` or ```nginx_stream_configs``` is not empty instead of checking nginx_official_repo_mainline (user is responsable of misconfiguration like that).
You can made variable "nginx_official_repo_mainline" default to "nginx_stream_params != [] or nginx_stream_configs != [] " in other hand.